CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 * Fixed an error that occurred when setting the `Waypoint.separatesLegs` property to `true` and setting the `Waypoint.targetCoordinate` property. ([#480](https://github.com/mapbox/mapbox-directions-swift/pull/480))
 * `Directions.fetchAvailableOfflineVersions(completionHandler:)` now calls its completion handler on the main queue consistently. ([#475](https://github.com/mapbox/mapbox-directions-swift/pull/475))
 * Upgraded to Polyline v5.0.0. ([#487](https://github.com/mapbox/mapbox-directions-swift/pull/487))
+* Added `Intersection.regionCode` and `RouteStep.administrativeRegionIndicesByIntersection` for more convenient region data usage. `Intersection.administrativeRegionIndex` is now internal. ([#485](https://github.com/mapbox/mapbox-directions-swift/pull/485))
+* Added `Intersection.regionCode` and `RouteStep.administrativeRegionIndicesByIntersection` for more convenient region data usage. `Intersection.administrativeRegionIndex` is removed in favor of the above, depending on needs in a string representation or index value. ([#485](https://github.com/mapbox/mapbox-directions-swift/pull/485))
```

Comment:
#483 effectively removed these properties from the public API (as far as documentation is concerned), and we unreleased v1.1.0, so there’s no need to talk about the APIs we briefly exposed prior to release.

Reply:
Removed these lines